modules/Layout/CookieConsentBar/CookieConsentBar.module.scss

```diff
@@ -94,3 +94,9 @@
         width: 100%;
     }
 }
+
+.acceptButton {
+    &:hover {
```

Comment:
Maybe applying this for `:active` and `:focus` states too?

Reply:
thanks, applied